flow/view_slicer.cc

```diff
@@ -0,0 +1,116 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/flow/view_slicer.h"
+
+#include <unordered_map>
+#include "flow/embedded_views.h"
+#include "fml/logging.h"
+
+namespace flutter {
+
+std::unordered_map<int64_t, SkRect> SliceViews(
+    DlCanvas* background_canvas,
+    const std::vector<int64_t>& composition_order,
+    const std::unordered_map<int64_t, std::unique_ptr<EmbedderViewSlice>>&
+        slices,
+    const std::unordered_map<int64_t, SkRect>& view_rects) {
+  std::unordered_map<int64_t, SkRect> overlay_layers;
+
+  auto current_frame_view_count = composition_order.size();
```

Comment:
just so that i understand - this count is the count of platform views right?

Reply:
Correct